apps/desktop-wallet/src/storage/assets/fungibleTokensSlice.ts

```diff
@@ -16,23 +16,25 @@ You should have received a copy of the GNU Lesser General Public License
 along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
-import { AssetInfo } from '@alephium/shared'
+import { FungibleToken } from '@alephium/shared'
 import { ALPH } from '@alephium/token-list'
 import { createSlice, EntityState } from '@reduxjs/toolkit'
 
-import { loadingStarted, syncNetworkTokensInfo, syncUnknownTokensInfo } from '@/storage/assets/assetsActions'
-import { assetsInfoAdapter } from '@/storage/assets/assetsAdapter'
+import { syncUnknownTokensInfo, syncVerifiedFungibleTokens } from '@/storage/assets/assetsActions'
+import { fungibleTokensAdapter } from '@/storage/assets/assetsAdapter'
 import { customNetworkSettingsSaved, networkPresetSwitched } from '@/storage/settings/networkActions'
 
-interface AssetsInfoState extends EntityState<AssetInfo> {
-  loading: boolean
+interface FungibleTokensState extends EntityState<FungibleToken> {
+  loadingVerified: boolean
+  loadingUnverified: boolean
   status: 'initialized' | 'uninitialized'
-  checkedUnknownTokenIds: AssetInfo['id'][]
+  checkedUnknownTokenIds: FungibleToken['id'][]
 }
 
-const initialState: AssetsInfoState = assetsInfoAdapter.addOne(
-  assetsInfoAdapter.getInitialState({
-    loading: false,
+const initialState: FungibleTokensState = fungibleTokensAdapter.addOne(
+  fungibleTokensAdapter.getInitialState({
+    loadingVerified: false,
+    loadingUnverified: false,
```

Comment:
Splitting this into 2 allows us to display data faster, which is basically replicating the experience in our latest release.